plot.py

```diff
@@ -397,7 +403,7 @@ def plot(self, df_in, output_prefix, output_dir, dpi, force, show, verbose):
                 y=self.ykey, 
                 hue=self.huekey, 
                 style=self.stylekey, 
-                markers=True,
+                markers=False,
```

Comment:
With markers disabled series are no longer distinguishable (colour alone is insufficient for visually impaired viewers / black and white)

Reply:
fixed